### PR TITLE
feat: Add IconComponent

### PR DESCRIPTION
**Pull request by KonTra7cf**

### CODE REVIEW

**Review by wikbro7650 — APPROVED** [review]

File: src/components/atoms/IconComponent/Icon.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    user: User
+};
+
+export const Icon = ({ iconName, size, className, onClick }) => {
+    const handleClick = useCallback(() => onClick(), [onClick]);
+    const TheIcon = icons[iconName];
+
+    return (
+        <TheIcon
+            className={`icon icon${size} icon${className}`}
```
Comment (Veryyapeee) [inline src/components/atoms/IconComponent/Icon.jsx:40]: Missing __ in class names

File: src/components/atoms/IconComponent/Icon.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    home: Home,
+    logo: Logo,
+    power: Power,
+    send: Send,
+    user: User
+};
+
+export const Icon = ({ iconName, size, className, onClick }) => {
+    const handleClick = useCallback(() => onClick(), [onClick]);
+    const TheIcon = icons[iconName];
```
Comment (Veryyapeee) [inline src/components/atoms/IconComponent/Icon.jsx:36]: Name it just "Icon"

**Review by Veryyapeee — CHANGES_REQUESTED** [review]
Looks cool, minor comments

File: src/components/atoms/IconComponent/Icon.jsx (hunk 2 above)
Comment (KonTra7cf) [reply src/components/atoms/IconComponent/Icon.jsx:36]: error  'Icon' is already declared in the upper scope on line 34 column 14  no-shadow. - ESLint does not want me to rename it to "Icon"

**Review by PoProstuMieciek — APPROVED** [review]

**Review by Veryyapeee — APPROVED** [review]

**Review by Mariusz-web — APPROVED** [review]

**Pull request merged**